src/bundle/Resources/public/js/scripts/admin.notifications.list.js

```diff
@@ -0,0 +1,176 @@
+(function (global, doc, ibexa, Translator, Routing) {
+    const SELECTOR_MODAL_ITEM = '.ibexa-notifications-modal__item';
+    const SELECTOR_GO_TO_NOTIFICATION = '.ibexa-notification-view-all__show';
+    const SELECTOR_TOGGLE_NOTIFICATION = '.ibexa-notification-view-all__mail';
+    const { showErrorNotification } = ibexa.helpers.notification;
+    const { getJsonFromResponse } = ibexa.helpers.request;
+    const markAllAsReadBtn = doc.querySelector('.ibexa-notification-list__btn--mark-all-as-read');
+    const markAsReadBtn = doc.querySelector('.ibexa-notification-list__btn--mark-as-read');
+    const deleteBtn = doc.querySelector('.ibexa-notification-list__btn--delete');
+    const notificationsCheckboxes = [
+        ...doc.querySelectorAll('.ibexa-notification-list .ibexa-table__cell--has-checkbox .ibexa-input--checkbox'),
+    ];
+    const markAllAsRead = () => {
+        const markAllAsReadLink = Routing.generate('ibexa.notifications.mark_all_as_read');
+
+        fetch(markAllAsReadLink, { mode: 'same-origin', credentials: 'same-origin' })
+            .then(getJsonFromResponse)
+            .then((response) => {
+                if (response.status === 'success') {
+                    global.location.reload();
+                }
```

Comment:
Shouldn't we handle other responses? Like forbidden etc, catch is only when fetch fails, so there still needs to be some handler for other status codes (mainly 4xx and 5xx) I think